loader/trampoline.c

```diff
@@ -2542,7 +2542,7 @@ VkResult setupLoaderTrampPhysDevGroups(VkInstance instance, struct loader_instan
     }
 
     // Call down and get the content
-    fpEnumeratePhysicalDeviceGroups(instance, &total_count, local_phys_dev_groups);
+    fpEnumeratePhysicalDeviceGroups(inst->instance, &total_count, local_phys_dev_groups);
```

Comment:
The only thing that could make this code better is that the function removes the `instance` parameter, making it impossible to use the wrong type. I didn't see anywhere else that it was used in `setupLoaderTrampPhysDevGroups` but thats more of a nitpick.

Reply:
Thanks!  I think there are lots of things that _could_ make the existing code better, like removing the double call that this function causes for every single app call to EnumeratePhysicalDeviceGroups.  But #770 already does that.  Your suggestion is reasonable, but I think #831 is the simplest, clearest change that solves this problem for any layer or driver that wraps or keys from the `VkInstance` parameter and it should be easy to justify cherry-picking.  I suggest we merge this and then focus on #770, which cleans up this functionality as a whole.